apps/src/lib/ui/lti/link/LtiLinkAccountPage/WelcomeBanner/index.tsx

```diff
@@ -1,54 +1,37 @@
 import styles from '@cdo/apps/lib/ui/lti/link/LtiLinkAccountPage/link-account.module.scss';
-import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon';
 import Typography from '@cdo/apps/componentLibrary/typography/Typography';
-import React, {useCallback, useContext} from 'react';
+import React, {useContext} from 'react';
 import i18n from '@cdo/locale';
 
-import canvas from '../../../assets/canvas.svg';
-import schoology from '../../../assets/schoology.svg';
 import codeLogo from '../../../assets/codeLogo.svg';
 import {LtiProviderContext} from '../context';
 
 const WelcomeBanner = () => {
-  const {ltiProvider, ltiProviderName} = useContext(LtiProviderContext)!;
-
-  const getLtiProviderIcon = useCallback(() => {
-    switch (ltiProvider) {
-      case 'canvas_cloud':
-      case 'canvas_beta_cloud':
-      case 'canvas_test_cloud':
-        return canvas;
-      case 'schoology':
-        return schoology;
-      default:
-        return undefined;
-    }
-  }, [ltiProvider]);
-
-  const ltiProviderIcon = getLtiProviderIcon();
```

Comment:
Is it intentional for the LMS <-> Code.org images to be removed?

Reply:
Yes, this was part of the Figma but I can definitely preserve this with going the route of splitting this up

Reply:
@moshebaricdo Just want to confirm that this is to be intentionally removed. If so, I think we can remove the icons from the repo as well.

Reply:
Yes, the LMS view should preserve the two icons.

Reply:
Oh I think I misunderstood the Figma, so we want to keep the arrows and the icons for the LMS view? I thought we were removing them alltogether

Reply:
Ah yes, sorry, there is a separate frame for the LMS version in the mocks! There should be some redlines for spacing between them

Reply:
Oh I see now, no my bad for not seeing the separation there. I'll fix that then!